lib/ca_certs.ml

```diff
@@ -4,13 +4,14 @@ let rec detect_list =
   function
   | [] -> return_none
   | path :: paths ->
-      Lwt_unix.file_exists path >>= fun exists ->
-      if exists then return_some (`Ca_file path) else detect_list paths
+    Lwt_unix.file_exists path
+    >>= fun exists -> if exists then return_some (`Ca_file path) else detect_list paths
+;;
 
 let locations =
-  [
-    "/etc/ssl/certs/ca-certificates.crt";
-    "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem";
+  [ "/etc/ssl/certs/ca-certificates.crt"
+  ; "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```

Comment:
I know that this is quite widely used, but what is reasoning behind formatting like this? Personally I find this really weird.

Reply:
I mean here that the delimiter is added before next item.

Reply:
I guess it is more consistent syntactically with ADTs and records.